IdentityCore/src/requests/MSIDSilentTokenRequest.m

```diff
@@ -202,6 +215,10 @@ - (void)executeRequestImpl:(MSIDRequestCompletionBlock)completionBlock
             {
                 MSID_LOG_WITH_CTX(MSIDLogLevelInfo, self.requestParameters, @"Cached key thumbprint is different from access token's key thumbprint, removing it..");
             }
+            else if(self.unexpiredRefreshNeededAccessToken)
+            {
+                MSID_LOG_WITH_CTX(MSIDLogLevelInfo, self.requestParameters, @"Access token refresh In expired, removing it...");
```

Comment:
`Access token refresh In expired, removing it..`

Is this correct log statement?

Reply:
You are right , thi was unwanted logging, removed